platform/os_linux.go

```diff
@@ -132,17 +161,19 @@ func (p *execClient) ClearNetworkConfiguration() (bool, error) {
 	return false, nil
 }
 
+// not supported on linux
 func (p *execClient) ExecutePowershellCommand(_ string) (string, error) {
```

Comment:
not your problem to fix, but I really hate that this has to be stubbed for linux...this interface/platform impls arch could be improved